### ISSUE TITLE
Keep branches highlighted when filtering date range, i.e. filter only tips not branches

### ISSUE THREAD

**Issue by corneliusroemer** [nextstrain/auspice#1405]
It would be nice if branches were not filtered out by the date range filter. I would like the behaviour to be similar to normal filters. When I filter for mutations for example, I can still see the branches highlighted that lead to tips. This allows me to easily select these branches. See screenshot:
![image](https://user-images.githubusercontent.com/25161793/134736654-76088abc-071b-46ff-901b-6de15ad9081d.png)

With dates, it's different. No branches remain visible and thus no branches can be clicked, which is a shame (I think):
![image](https://user-images.githubusercontent.com/25161793/134736657-d065bd72-88fe-43d6-baa1-1511d29b13b4.png)

**Comment by jameshadfield** [comment]
Genotype filtering currently won't always highlight branches the same way as "normal" filters (i.e. filters which act on other metadata). For instance, see https://nextstrain.org/zika/?c=gt-NS5_267&gt=NS5.267A. 
![image](https://user-images.githubusercontent.com/8350992/134743398-6c9397ee-466d-43be-b429-6a2ef0d7a82e.png)

This is being debated in https://github.com/nextstrain/auspice/issues/1281.

If we filter time to `min=2021` then wouldn't it be strange to have branches from 2020 highlighted? (It's a bit more complicated than that of course, because a branch may span the time cutoff).

**Comment by corneliusroemer** [comment]
Yes I think the last screenshot makes sense. We see branches that lead to tips that are still active. Don't you think this makes sense? Otherwise I can't click anywhere.

**Comment by jameshadfield** [comment]
Yup - but this is the current behavior (e.g. https://nextstrain.org/zika/?dmin=2016-09-20). nCoV is the same, it's just that there are so many tips that the obscure a lot of the branches behind them!

**Comment by corneliusroemer** [comment]
Dang, you're so right! I was just too zoomed out. Now I can see it.
![Uploading image.png…]()

**Comment by huddlej** [comment]
> It would be nice if branches were not filtered out by the date range filter. I would like the behaviour to be similar to normal filters.

I feel exactly the same way. I understand the logical argument for showing the most accurate representation of the tree for a given set of filters, but as a end-user I just want to be able to click on internal nodes that precede the min date, so I can zoom to specific regions of the tree. I think there is a tension here between accurately representing data in the tree that matches the filters and disabling a functional interface that users rely on (clicking branches to zoom).

**Comment by corneliusroemer** [comment]
True, I'm still not able to click on the mother branches. I think the filter should just act on tips, not on branches. Branches should always show if there are tips from that branch.

I will reopen this issue to that end.
